libcontainer/configs/validate/rootless.go

```diff
@@ -91,27 +68,3 @@ func rootlessCgroup(config *configs.Config) error {
 
 	return nil
 }
-
-// mount verifies that the user isn't trying to set up any mounts they don't have
-// the rights to do. In addition, it makes sure that no mount has a `uid=` or
-// `gid=` option that doesn't resolve to root.
-func rootlessMount(config *configs.Config) error {
```

Comment:
I'm not convinced that dropping this makes sense. Even in the multiple-uid case, really we should be making it so that we emit an error if the `uid=...` or `gid=...` options reference unmapped users. The errors are far too cryptic if you ignore it.